### PR TITLE
s/controller = null;/controller;

### LINKED ISSUE #236: Address nullability of DisplayMediaStreamOptions.controller
I believe we should make the new `CaptureController controller` member in the `DisplayMediaStreamOptions`  dictionary either:
- optional with `CaptureController? controller = null;`- even though it is already optional according to https://webidl.spec.whatwg.org/#dictionary-member-optional, 
- or remove the `= null;` part that was added in https://github.com/w3c/mediacapture-screen-share/pull/235/commits/73770e40390ebc25fbad80a5a7171da53d487afc

https://github.com/w3c/mediacapture-screen-share/blob/7857efe4c25628af2e25fca5eda9ab93927c9bd3/index.html#L913

### PR DESCRIPTION
**Pull request by eladalon1983**
Fixes #236.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/mediacapture-screen-share/pull/237.html" title="Last updated on Sep 30, 2022, 2:46 PM UTC (9bd6536)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/mediacapture-screen-share/237/7857efe...9bd6536.html" title="Last updated on Sep 30, 2022, 2:46 PM UTC (9bd6536)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -910,7 +910,7 @@ <h2>DisplayMediaStreamOptions</h2>
 dictionary DisplayMediaStreamOptions {
   (boolean or MediaTrackConstraints) video = true;
   (boolean or MediaTrackConstraints) audio = false;
-  CaptureController controller = null;
+  CaptureController controller;
```
Comment (beaufortfrancois) [inline index.html:913]: You may want to change section code below as well:

```diff
-                  <dfn><code>controller</code></dfn> of type <code>{{CaptureController}})</code>,
-                  defaulting to <code>null</code>
+                  <dfn><code>controller</code></dfn> of type <code>{{CaptureController}}</code>
```

```diff
-                      If not <code>null</code>, this {{CaptureController}} object will be
+                      If present, this {{CaptureController}} object will be
```

File: index.html [hunk 2]
```diff
@@ -944,14 +944,13 @@ <h2>Dictionary <a class="idlType">DisplayMediaStreamOptions</a>
                   will not contain an audio track.</p>
                 </dd>
                 <dt>
-                  <dfn><code>controller</code></dfn> of type <code>{{CaptureController}})</code>,
-                  defaulting to <code>null</code>
+                  <dfn><code>controller</code></dfn> of type <code>{{CaptureController}})</code>
```
Comment (beaufortfrancois) [inline index.html]:
```suggestion
                  <dfn><code>controller</code></dfn> of type <code>{{CaptureController}}</code>
```
image

**Comment by jan-ivar** [comment]
I disagree with this, explanation forthcoming.

**Comment by jan-ivar** [comment]
I think https://github.com/w3c/mediacapture-screen-share/pull/238 is the right fix instead here. I'll comment on why in https://github.com/w3c/mediacapture-screen-share/issues/236.

**Review by jan-ivar — APPROVED** [review]

**Review by youennf — APPROVED** [review]

**Pull request merged**